### PR TITLE
Feature/cxspa 349

### PR DESCRIPTION
**Pull request by psmul**

### CODE REVIEW

File: feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.spec.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      `addCartEntryComment-user1-quoteCode1-entryNumber1-${comment.toString()}`
+    );
+    expect(adapter.addCartEntryComment).toHaveBeenCalledWith(
+      'user1',
+      'quoteCode1',
+      'entryNumber1',
+      comment
+    );
+  });
+});
```
Comment (pawelfras) [inline feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.spec.ts:210]: `could:` consider to move to constants all values that are used during service calls and assertions and reuse them where needed (applies to all tests cases in this file)

File: feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.ts [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  QuoteStarter,
+  Comment,
+} from '../model/commerce-quotes.model';
+import { CommerceQuotesAdapter } from './commerce-quotes.adapter';
+
+@Injectable()
+export class CommerceQuotesConnector {
+  constructor(protected adapter: CommerceQuotesAdapter) {}
+
+  public getQuotes(userId: string): Observable<QuoteList> {
```
Comment (pawelfras) [inline feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.ts]: `query:` as I see this method is not supporting features like pagination and sorting. Should it be handled here or during Quote list implementation?

File: feature-libs/commerce-quotes/core/model/commerce-quotes.model.ts [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+import { Price, PaginationModel, Principal } from '@spartacus/core';
+
+export interface Quote {
+  allowedActions?: string[];
+  cartId?: string;
+  code: string;
+  comments?: Comment[];
+  creationTime?: Date;
+  description?: string;
+  entries?: OrderEntry[];
```
Comment (pawelfras) [inline feature-libs/commerce-quotes/core/model/commerce-quotes.model.ts:11]: `should:` `OrderEntry` interface doesn't contain fields like `comments` and `configurationInfos` so it has to be  extended

File: feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts [hunk 4]
```diff
@@ -0,0 +1,312 @@
+// import { HttpClient } from "@angular/common/http";
```
Comment (pawelfras) [inline feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts]: `should:` remove comment

File: feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts [hunk 5]
```diff
@@ -0,0 +1,312 @@
...
+        addCartEntryComment:
+          '/users/${userId}/quotes/${quoteCode}/entries/${entryNumber}/comments',
+      } as OccEndpoints,
+    },
+  },
+};
+
+describe(`OccCheckoutDeliveryModesAdapter`, () => {
+  let service: OccCommerceQuotesAdapter;
+  // let httpClient: HttpClient;
```
Comment (pawelfras) [inline feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts]: `should:` remove comment

Comment (psmul) [reply feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts]: already removed with [ef44817](https://github.com/SAP/spartacus/pull/15742/commits/ef44817571204b0b95783311a669079033338612) commit

File: feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts (hunk 4 above)
Comment (psmul) [reply feature-libs/commerce-quotes/occ/adapters/occ-commerce-quotes.adapter.spec.ts]: already removed with [ef44817](https://github.com/SAP/spartacus/pull/15742/commits/ef44817571204b0b95783311a669079033338612) commit

File: feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.spec.ts (hunk 1 above)
Comment (psmul) [reply feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.spec.ts:210]: Done. Thanks for pointing this out!

File: feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.ts (hunk 2 above)
Comment (psmul) [reply feature-libs/commerce-quotes/core/connectors/commerce-quotes.connector.ts]: Actually nice catch, it's mu omission and I think it should be already prepared at this stage

**Review by pawelfras — APPROVED** [review]

**Pull request merged**